ahnlich/ai/src/cli/server.rs

```diff
@@ -94,6 +94,12 @@ pub struct AIProxyConfig {
     /// List of ai models to support in your aiproxy stores
     #[arg(long, required(true), value_delimiter = ',')]
     pub(crate) supported_models: Vec<SupportedModels>,
+
+    /// AI Model Idle Time in seconds
+    /// Time in seconds for an unused/idle supported model to be frozen
```

Comment:
Not frozen. It gets dropped completely. Also add that calling it after drop reinitializes the model from scratch